test/functional/wallet_createwallet.py

```diff
@@ -96,5 +96,23 @@ def run_test(self):
         assert_raises_rpc_error(-4, "Error: This wallet has no available keys", w5.getnewaddress)
         assert_raises_rpc_error(-4, "Error: This wallet has no available keys", w5.getrawchangeaddress)
 
+        self.log.info('Test creating a new encrypted wallet.')
+        assert_raises_rpc_error(-4, 'Cannot create an encrypted empty wallet.', self.nodes[0].createwallet, 'w6', False, True, 'thisisapassphrase')
+        # Encrypted wallet is created
+        self.nodes[0].createwallet('w6', False, False, 'thisisapassphrase')
+        w6 = node.get_wallet_rpc('w6')
+        w6.walletpassphrase('thisisapassphrase', 10)
+        # There should only be 1000 keys
```

Comment:
Nit: 1 key (and explain that normally encrypting a wallet adds new keys)

Reply:
fine